tooling/karaf-maven-plugin/pom.xml

```diff
@@ -372,8 +372,9 @@
                 <groupId>org.apache.maven.plugins</groupId>
                 <artifactId>maven-compiler-plugin</artifactId>
                 <configuration>
-                    <source>1.8</source>
-                    <target>1.8</target>
+                    <release>21</release>
```

Comment:
We can remove this part of it's already defined by the `maven.compiler` properties.